packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -457,4 +474,268 @@ linter:
       printStatus('No analyzer warnings! (ran in ${elapsed}s)');
     return 0;
   }
+
+  Future<int> _analyzeWatch() async {
+    List<String> directories;
+
+    if (isFlutterRepo) {
+      printStatus('Analyzing Flutter repository:');
+      directories = <String>[];
+      directories.addAll(_gatherProjectPaths(path.absolute('examples')));
+      directories.addAll(_gatherProjectPaths(path.absolute('packages')));
+      for (String projectPath in directories)
+        printStatus('  ${path.relative(projectPath)}');
```

Comment:
I'll tighten it up; I was assuming that the flutter repo would be a relatively rare use case.